release-notes/7.0/preview/api-diff/preview1/Microsoft.NETCore.App/7.0-preview1_System.Runtime.Intrinsics.md

```diff
@@ -0,0 +1,412 @@
+# System.Runtime.Intrinsics
```

Comment:
System.Runtime.Intrinsics LGTM

Reply:
CC. @dotnet/area-system-runtime-intrinsics 

Also see Arm/Arm64 and x86/x64 specific intrinsics above